### PR TITLE
Add nvenergy smart thermostat support

### PR DESCRIPTION
**Pull request by vegascom-jeff**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @vegascom-jeff, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#453).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/nvenergy.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "NV Energy Smart Thermostat",
+  "owner": ["@vegascom-jeff"],
+  "manifest": "https://raw.githubusercontent.com/vegascom-jeff/nvenergy/main/manifest.json",
```
Comment (nergal) [inline components/nvenergy.json:4]: @vegascom-jeff I'm not a maintainer for this repo, just passing by - however, I've noticed that the manifest.json file has a syntax error (no comma after `issue_tracker` section) in the current revision (`f464722`).

**Review by vegascom-jeff — COMMENTED** [review]
Thank you!

**Review by frenck — COMMENTED** [review]
This integration has the following in its manifest:

```
"requirements": ["requests", "cryptography"],
```

However, those are already core dependencies and don't need to be added.

See: https://developers.home-assistant.io/docs/creating_integration_manifest/#custom-integration-requirements

The suggestion is to remove those 2 requirements from the integration manifest.

With those two removed, there are no requirements left, meaning there are no wheels to build for this custom integration and thus this integration doesn't need to be added to this repository.

../Frenck

**Comment by vegascom-jeff** [comment]
Thanks @frenck , I missed that in the docs. Still wrapping my head around everything!

**Pull request closed without merging**